.github/workflows/run-tests.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       mina-local-network:
-        image: o1labs/mina-local-network:o1js-main-latest-lightnet
+        image: o1labs/mina-local-network:compatible-latest-lightnet
```

Comment:
`o1labs/mina-local-network:o1js-main-latest-lightnet` is not a valid image name, see: https://github.com/o1-labs/Archive-Node-API/actions/runs/11448838391/job/31853195062